docs/modules/help-support-troubleshooting/pages/log-information.adoc

```diff
@@ -0,0 +1,20 @@
+
+= Trace {page-component-title} Logs
+
+This section describes the log files that {page-component-title} creates, and what to look for in them when troubleshooting.
```

Comment:
```suggestion
This section describes common log files to look at when troubleshooting.
```

Reply:
I don't think we need to cover _every_ log file here - just the main ones when troubleshooting.  The References:Daemons section does mention log files associated with each daemon if users are needing to delve into more specific troubleshooting.

Reply:
It currently shows only one log file, but change Mark suggests allows for that.